docs/release_notes/v0.16.0-alpha.md

```diff
@@ -0,0 +1,109 @@
+
+## Highlights in v0.16-alpha
+
+- **Secret Stores**: More than one [Secret Store](https://docs.spiceai.org/components/secret-stores) can now be specified. For example, to configure Spice with both [Environment Variable](https://docs.spiceai.org/components/secret-stores/env) and [AWS Secrets Manager](https://docs.spiceai.org/components/secret-stores/aws-secrets-manager) Secret Stores, use the following `secrets` configuration in `spicepod.yaml`:
+
+```yaml
+secrets:
+  - from: env
+    name: env
+  - from: aws_secrets_manager:my_secret_name
+    name: aws_secret
+```
+
+Secrets managed by configured Secret Stores can be referenced in component `params` using the syntax `${<store_name>:<key>}`. E.g.
+
+```yaml
+datasets:
+  - from: postgres:my_table
+    name: my_table
+    params:
+      pg_host: localhost
+      pg_port: 5432
+      pg_password: ${{ env:MY_PG_PASS }}
+```
+
+- **Java Client SDK**: The [Spice Java SDK](https://github.com/spiceai/spice-java) has been released for JDK 17 or greater.
+
+- **Federated SQL Query**: Significant stability and reliability improvements have been made to federated SQL query support in most data connectors.
+
+## Breaking Changes
+
+- **Secret Stores**: Secret Stores support has been overhauled including required changes to `spicepod.yml` schema. File based secrets stored in the `~/.spice/auth` file are no longer supported. See [Secret Stores Documentation](https://docs.spiceai.org/components/secret-stores) for full reference.
+
+To upgrade Secret Stores, change:
+
+```yaml
+
+```
+
+to:
+
+```yaml
+
+```
+
+- **Datasets**: The default value of [`time_format`](https://docs.spiceai.org/reference/spicepod/datasets#time_format) has changed from `unix_seconds` to `timestamp`.
+
+To upgrade:
+
+```yaml
+datasets:
+  - from:
+    name: my_dataset
+    # Explicitly define format when not specified.
+    time_format: unix_seconds
+```
+
+- **HTTP Port**: The default HTTP port has changed from port `3000` to port `8090` to avoid conflicting with frontend apps which typically use the 3000 range. If an SDK is used, upgrade it at the same time as the runtime.
+
+To upgrade and continue using port 3000, run spice with the port command:
+
+```shell
+# Via CLI
+spice run --port 3000
+
+# Using Dockerfile or spiced directly
+spiced --port 3000
+```
+
+- **HTTP Metrics Port**: The default HTTP Metrics port has changed from port `9000` to `9090` to avoid conflicting with other metrics protocols which typically use port 9000.
+
+To upgrade and continue using port 9000, run spice with the port command:
+
+```shell
+# Via CLI
+spice run --metrics-port 9000
+
+# Using Dockerfile or spiced directly
+spiced --metrics-port 9000
+```
```

Comment:
Overriding the metrics port via CLI isn't supported.

The metrics port isn't set by default on the runtime itself, its only set from the CLI when calling `spice run`. Calling `spiced` directly isn't a breaking change, since they would have had to set the port explicitly anyway.